### ISSUE TITLE
Starte og drive: Inngang til tjenesten rett fra start

### ISSUE THREAD

**Issue by alisekjelling** [Altinn/altinn-infoportal-optimizely#141]
Brønnøysundregistrene / Starte og drive skal gi ut en innlogget tjeneste som gir tilpasset veiledning til virksomheter - i hovedsak ENK og AS under 10 personer. 

Vi må sammen finne ut av hvordan inngangen til denne tjenesten skal være. Vi starter med et møte før jul med Irene, Håvard, Frode og Lene. 

Målet er at tjenesten skal lanseres i prod til alle i jan/feb 2024.

-----

Diskusjon 04.11

- Inngang til tjenesten kan være under kontakt og hjelp
- I et fremtidsperspektiv: Se på konseptet starte og drive og om vi kan øke forståelsen og kjennskapen til hva dette er ute hos de som trenger det, men ikke vet det finnes.

**Comment by alisekjelling** [comment]
Planlagt at vi tar møte med Lene på nyåret.

**Comment by dortedrange** [comment]
Til gjennomgang 12.11: vi hørte vel ikke mer fra BRREG om dette, de ordnet det på egenhånd. Så kan vel settes til done?

**Comment by dortedrange** [comment]
@ferkinx Er du enig i at vi enten sletter eller setter denne til done?

**Comment by ferkinx** [comment]
@dortedrange Ja, det ble bare en stille lansering - og de ble vel aldri helt ferdig. Jeg setter den til done :) 

Vi får heller komme tilbake til saken hvis det skjer noe mer - for dette er vel i realiteten BR og Skatt sin leveranse på livshendelsen starte og drive.
